### PR TITLE
mod-wrapper

### PR DESCRIPTION
**Pull request by KhanhHunq**
done task mod-wrapper

### CODE REVIEW

**Comment by thinhvu2801** [comment]
okay

File: mod-banner-2.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+                                              <!-- mod-wrapper -->
+      <section class="module mod-banner-2">
+    <div class="container">
+      <div class=" text-center text-white py-50 xl:px-6  md:px-18 bg-blue-400 rounded font-plus">
+      <div class="head-content h1 text-4xl mb-20 leading-normal md:px-40 px-15">
+        <p>10 million + adjustable bases sold worldwide</p>
+      </div>  
+        <div class="row text-base font-normal px-30  ">
+          <div class=" w-full md:w-1/2 lg:w-1/4 col1 ">
```
Comment (ghuy2003) [inline mod-banner-2.html:9]: 4 columns is well done !
very beautiful 
perfecto
TEST DONE !

**Comment by ghuy2003** [comment]
README check: "null"

**Comment by Araxys123** [comment]
need some improvements but otherwise okay, nice work

**Pull request merged**